addins/Cake.NuGet.yml

```diff
@@ -0,0 +1,11 @@
+Name: Cake.NuGet
```

Comment:
Should be filtered as it is a module

Reply:
uh??? A module that does not follow the naming convention

Reply:
If you have that information I think you can filter out anything from the [cake-build](https://www.nuget.org/profiles/cake-build) user on nuget.org, which would also filter this one.

Reply:
Should we audit Cake.Nuget even though it’s shipped by the Cake team or should we exclude it?

Reply:
Cake.NuGet ships with each release of Cake, i.e. it is one of the NuGet packages that we push every time we release Cake, so it doesn't need to be audited.

Reply:
Done. Cake.NuGet added to blacklist